guide/README.rst

```diff
@@ -12,40 +12,20 @@ Installation
 Building VOTCA
 ^^^^^^^^^^^^^^
 
-The simplest way is to use CMake:
-
-.. code:: bash
-
-    prefix=~/votca
-    version=master # or 'stable' or 'v1.4.1'
-    git clone -b ${version} --recursive https://github.com/votca/votca.git
-    cd votca
-    mkdir build
-    cd build
-    cmake -DCMAKE_INSTALL_PREFIX=${prefix} ..
-    make -j5
-    make install
-
-To build a gromacs version for VOTCA use
-
-.. code:: bash
-
-    cmake -DBUILD_OWN_GROMACS=ON ..
-    make -j5
-    make install
+Instructions how to build and install VOTCA can be found
+`here <https://www.votca.org/INSTALL.html>`__
```

Comment:
@JensWehner @felipeZ how would I use `:ref:` here instead of direct link?

Reply:
I guess we need labels: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref

Reply:
Ahh it should stay inside the manual.

Reply:
You can also use references like the following:
```rst
Then  install RDKit_ and H5PY_:
  - `conda install -y -q -c conda-forge h5py rdkit`


.. _RDKit: https://www.rdkit.org
.. _H5PY: https://www.h5py.org/
```

The underscore at the end of the keyword instructs rst to generate a link to the corresponding link. The last two lines are just shown, just the references.

Reply:
@marvinbernhardt do know how to do this?

Reply:
Adding 
```
+.. _install:
+
 Installation
 ============
```
at the top of votca/share/doc/INSTALL.rst and changing the link here to
```
 Instructions how to build and install VOTCA can be found
-`here <https://www.votca.org/INSTALL.html>`__
+:ref:`here<install>`.
```
works for me. Can I not commit to this branch or do I have my local settings mixed up?